src/PIL/ImageWin.py

```diff
@@ -88,8 +88,8 @@ def expose(self, handle):
         Copy the bitmap contents to a device context.
 
         :param handle: Device context (HDC), cast to a Python integer, or an
-                       HDC or HWND instance.  In PythonWin, you can use the
-                       :py:meth:`CDC.GetHandleAttrib` to get a suitable handle.
```

Comment:
It would be ideal to link to PythonWin docs for the function, but I can't find them, so this is probably fine.